ToDo.DataAccess/ToDo.DataAccess/Repositories/CustomRepositories/ItemRepository.cs

```diff
@@ -1,23 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Text;
 using System.Linq;
 using ToDo.DataAccess.DataBase;
 using ToDo.DataAccess.Models;
 using Microsoft.EntityFrameworkCore;
+using ToDo.DataAccess.Repositories.CustomRepositories;
 
 namespace ToDo.DataAccess.Repositories
 {
-    public interface IItemRepository : IGenericRepository<Item>
-    { 
-        
-    }
-
     public class ItemRepository : GenericRepository<Item>, IItemRepository
     {
         public ItemRepository(ToDoDbContext context) : base(context)
         {
+            ToDoDbInitializer.Initialize(context);
```

Comment:
WTF?

Reply:
I recreated base and forgot  to delete it.